README.md

```diff
@@ -172,6 +172,46 @@ TID is a 32-bit integer to identify threads created with `wasi_thread_spawn`.
     For example, it can be used to indicate the main thread, which doesn't
     have a TID in the current version of this proposal.
 
+### Process
+
+* A process is a group of threads.
+
+* The main thread starts with a process which only contains
+  the main thread.
+
+* Threads created by a thread in a process using `wasi_thread_spawn`
+  are added to the process.
+
+* When a thread is terminated, it's removed from the process.
+
+### Voluntary thread termination
+
+A thread can terminate itself voluntarily by returning from
+`wasi_thread_start`.
+
+### Changes to WASI `proc_exit`
+
+With this proposal, the `proc_exit` function takes extra responsibility
+to terminate all threads in the process, not only the calling one.
+
+Any of the threads in the process can call `proc_exit`.
+
+### Traps
+
+When a thread caused a trap, it terminates all threads in the process
```

Comment:
s/caused/causes/ ?

How about "When a trap occurs in any thread, the entire process is terminated."?

Reply:
> s/caused/causes/ ?

probably. i'm not good at english as you may noticed.

> 
> How about "When a trap occurs in any thread, the entire process is terminated."?

done.